### PR TITLE
Update in-memory-data-structure-store@M-EXrTDeAEMz_IkEi-ab4.md

### PR DESCRIPTION
**Pull request by kulterryan**
added content

### CODE REVIEW

File: src/data/roadmaps/redis/content/in-memory-data-structure-store@M-EXrTDeAEMz_IkEi-ab4.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# In-memory Data Structure Store
\ No newline at end of file
+# In-memory Data Structure Store
+
+An in-memory database is a purpose-built database that relies primarily on internal memory for data storage. It enables minimal response times by eliminating the need to access standard disk drives (SSDs). In-memory databases are ideal for applications that require microsecond response times or have large spikes in traffic, such as gaming leaderboards, session stores, and real-time data analytics. The terms main memory database (MMDB), in-memory database system (IMDS), and real-time database system (RTDB) also refer to in-memory databases.
+
+Read More on [https://aws.amazon.com/memorydb/](https://aws.amazon.com/memorydb/)
```
Comment (arikchakma) [inline src/data/roadmaps/redis/content/in-memory-data-structure-store@M-EXrTDeAEMz_IkEi-ab4.md:5]: Please check the [contribution guide](https://github.com/kamranahmedse/developer-roadmap/blob/master/contributing.md) for adding resources.

Comment (dansholds) [inline src/data/roadmaps/redis/content/in-memory-data-structure-store@M-EXrTDeAEMz_IkEi-ab4.md:5]:
```suggestion

Learn more from the following resources: 

- [@article@Amazon MemoryDB](https://aws.amazon.com/memorydb/)
```

**Comment by dansholds** [comment]
Stick to the contribution next time please

**Comment by kulterryan** [comment]
> Stick to the contribution next time please

Sure thing.

**Pull request merged**